azureadps-1.0/MSOnline/Set-MsolCompanySettings.md

```diff
@@ -38,8 +38,7 @@ This command turns on the self-serve password reset feature for all administrato
 ## PARAMETERS
 
 ### -SelfServePasswordResetEnabled
-Indicates whether to allow the use of the self-service password reset feature.
-This setting is applied company-wide.
+Indicates whether to allow the use of the self-service password reset feature for all administrators in the company.
```

Comment:
Isn't SSPR feature available for all users, not just the administrators? Please clarify.

Reply:
I have no idea who made this change or who you are waiting to hear back from.  This has nothing to do with the original PR I submitted, which is needed.